packages/app/src/cli/prompts/config.ts

```diff
@@ -73,11 +70,10 @@ export function validate(value: string): string | undefined {
   if (result.length > 238) return 'The file name is too long.'
 }
 
-export async function confirmPushChanges(options: PushOptions, app: App) {
+export async function confirmPushChanges(options: PushOptions, remoteConfiguration: AppConfiguration) {
```

Comment:
1. The `app` arg was no longer necessary after removing `mergeAppConfiguration`. 
2. There's already a `configuration` property in the `options` object. 
3. I wasn't sure if I should add `remoteConfiguration` as an additional arg or as an additional property in the `options` object. I chose to start by adding it as an additional arg to keep the code simpler while I focused on getting the tests green. 
4. If we chose to add `remoteConfiguration` to the `options` object instead of passing it as a second arg, should we rename `configuration` to something like `localConfiguration` so that it's more obvious that one property is the local config and the other property is the remote config?

Reply:
Hey @billfienberg, nice work with this refactor!. To me it seems to be fair to leave `remoteConfiguration` as a separate argument. There's no specific methodology about method arguments, except nesting them inside object if you have more than 3 or 4.
Regarding the `PushOptions` type, we usually add there the command flags or some other input prompted to the user in a previous step
@isaacroldan this PR looks good to me, what do you think?